pool/app/duplicate_contacts/service.ml

```diff
@@ -0,0 +1,48 @@
+let src = Logs.Src.create "user_import.service"
+let get_or_failwith = Pool_common.Utils.get_or_failwith
+let interval_s = 5
```

Comment:
Do we need to run this every 5 seconds? IMO also every hour would be fine.

Reply:
Yes, the command only looks for duplicates of one contact.